### DISCUSSION TITLE
Help: Camera not found after Synology NAS "soft" reset

### DISCUSSION THREAD

**Discussion by Grandma-Betty** [djdd87/SynoAI#100] [Q&A] [answered]
I had SynoAI running without any issues before.
Three days ago I had to "soft" reset my NAS, meaning a reset which keeps all data but DSM itself.
I have re-installed DSM 7.0.1 update 2 (the same version which I was on before the "soft" reset).
After istalling the Docker package, the old containers and settings were still there so I did not have to set up those again.
Only thing I had to re-install from scratch again was the Synology Surveillance Station package.
I have set up Surveillance Station with my one and only camera I have, which also is working fine now when using Sureveillance Station through the DSM webinterface. My camera is recoding and I can also get the live view.
So far, so good, everything on the NAS is working fine now, except SynoAI (yet).
Everytime I start the SynoAI container, it indeed starts, but then, after a few seconds, I get the following error message:
`GetCameras: The camera with the name 'camera01' was not found in the Surveillance Station camera list.`
I have already seen [this error message](https://github.com/djdd87/SynoAI#camera-names-with-spaces-in-cause-the-error-the-camera-with-the-name-my-camera-was-not-found) described on the SynoAI Github website, where it is mentioned to replace spaces with chars. But I don't have never any spaces within any of my URLs.
Also, the _action rule_ is giving me a green tick and deepstack is also set up correctly (I can reach the deepstack webinterface).

I have gone through things four times now and I cannot find anything wrong in my configuration. I have also tried to completely disable the firewall tempoarly but that did not help (it should not be the issue anyway as I have configured everything to communicate on the docker gateway IP address).

Again: Everything was running fine before the "soft" reset.

Could this be because somewhere in the background is still remainig data from the old SSS installation? I have set exactly the same camera device name ("camera01") which I had set before the "soft" reset. Do you think something could be the issue because of redundant data remaining in the background?

Thank you in advance!

EDIT: Seems to be solved by just downloading the latest docker image again. Strange thing though.

**Comment by djdd87** [comment, marked as answer]
Please raise an issue and when you do provide the logs, the screenshot of the camera config dialog showing the name and your config.
